explainer.md

```diff
@@ -22,7 +22,52 @@ The design process of the [Web Neural Network API](https://www.w3.org/TR/webnn/)
 
 With emerging ML innovations in both software and hardware ecosystem, one of the main challenges for the web is to bridge this software and hardware development and bring together a solution that scales across hardware platforms and works with any framework for web-based machine learning experiences. We propose the WebNN API as an abstraction for neural networks in the web browsers.
 
-![WebNN architecture](content/webnn_arch.png)
+```mermaid
+%%{ init : { "theme" : "neutral" }}%%
+flowchart TD
+
+  style Models stroke-width:0, fill:#fff
```

Comment:
With the hard coded white fill the page looks awkward but not unreadable in Dark Mode:

![image](https://github.com/user-attachments/assets/b3e1613f-87c2-491f-a19e-149328b09116)

I'm not familiar enough with mermaid to know if there's an easy fix.

Reply:
In theory you should be able to inject some CSS which applies to the whole background. As is I'm concerned this will be confusing.

Reply:
It looks like there's one combo (`"look" : "handDrawn", "theme" : "neo-dark"`) that produces a rather readable flowchart in dark mode. Based on Mermaid issue discussion, dark mode support has limitations. If someone has played with this more, please share your tips.

https://github.com/webmachinelearning/webnn/pull/847/commits/97bf5cf5364461d8d1b4b9a2a2b4f03afd70b02a